addons/l10n_kz/data/account_account_template_data.xml

```diff
@@ -0,0 +1,2017 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="a1000" model="account.account.template">
+        <field name="name">Денежные средства</field>
+        <field name="code">1000</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1100" model="account.account.template">
+        <field name="name">Краткосрочные финансовые активы</field>
+        <field name="code">1100</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1200" model="account.account.template">
+        <field name="name">Краткосрочная дебиторская задолженность</field>
+        <field name="code">1200</field>
+        <field name="reconcile" eval="True"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_payable"/>
```

Comment:
< field  name = " name " > Short-term receivables </ field >
        <field name="user_type_id" ref="account.data_account_type_payable"/> 

?

should be receivable

and the fact that it's among asset accounts should have ring a bell ;-)